d3a_interface/matching_algorithms/pay_as_clear_matching_algorithm.py

```diff
@@ -53,7 +54,7 @@ def get_matches_recommendations(self, matching_data):
             bids = data.get("bids")
             offers = data.get("offers")
             current_time = data.get("current_time")
-            clearing = self.get_clearing_point(bids, offers, current_time)
+            clearing = self.get_clearing_point(bids, offers, current_time, market_id)
```

Comment:
What is `market_id`? Is it a market slot ID, an asset ID, or something else?

Reply:
Its an area market slot id

Reply:
I'm more confused than before :D What's an "area market slot"? How's it different from a normal market slot?

Reply:
It's a `market slot ID`, I tried to be explicit in my last comment but apparently confuses you even more. What I meant was its area's market's slot ID

Reply:
We need to decide on this first IMO in order to not fall into the same pit as with the single and double quotes. I dont remember to rename `market` to `market_slot` . Until we decided on this, I vote vor keeping it consistently the old way where the `market_id` is the id of the multiple markets the area has (in the time dimension).

Reply:
@hannesdiedrich ok, let's postpone the conversation to the moment when we start the refactoring then.